### PR TITLE
Add custom value for coin drops when idle

### PR DESCRIPTION
**Pull request by michaelowens**
Allows the user to configure a different coin drop value for idlers/lurkers. Now with a clean commit...

Editor gets rid of unnecessary whitespace, that's why there's a few useless changes there.

### CODE REVIEW

File: plugins/coins/plugin.iced [hunk 1]
```diff
@@ -51,16 +52,13 @@ updateCoins = () =>
 						Viewer = new Mikuia.Models.Channel viewer
 						goAhead = true
 
-						if not rewardIdlers
-							if chatActivity[viewer]?[stream]?
-								if chatActivity[viewer][stream] == 0
-									goAhead = false
-							else
-								goAhead = false
+						idle = not chatActivity[viewer]?[stream] > 0
```
Comment (vendethiel) [inline plugins/coins/plugin.iced]: not sure that's what you mean. the `not` applies to `chatActivity[viewer]?[stream]`

**Comment by michaelowens** [comment]
Not sure where your comment went, @Maxorq , but I got it in my email as well. You're right, my coffee script is kind of rusty. Added brackets so the not operator doesn't just affect the first part of the condition.

**Comment by vendethiel** [comment]
It wasn't a comment from @Maxorq, it was for me. I removed it because it was wrong :). No brackets needed.

**Comment by michaelowens** [comment]
Oh my bad @vendethiel. Hmm, that's actually quite interesting as with brackets it takes the `> 0` into the not-operator, without brackets it doesn't. I'm not sure if there's a scenario where this could cause an issue without brackets, but so far I can't find any problems. With true/false/undefined/null values it seems to behalve properly without brackets, yes. I can remove them if you guys would prefer that.

**Comment by vendethiel** [comment]
Actually, seems like I was right in the first comment. duh.

```
$ iced -e "console.log (not 5 > 8)"
false
$ iced -e "console.log (not (5 > 8))"
true
$ iced -e "console.log (not 8 > 5)"
false
```

**Comment by michaelowens** [comment]
Ah, yes exactly. Obviously leaving them in there.

**Comment by vendethiel** [comment]
I should've thought a little bit more about it before removing the comment :). My bad.

**Comment by michaelowens** [comment]
No worries, I got notified by mail and realised you were right none the less. :smile:

**Comment by Maxorq** [comment]
not a way i would like to implement this

**Comment by michaelowens** [comment]
Care to explain why? Just wondering, 'cause if you're so against it I can imagine it has a reason.  :)

**Comment by Maxorq** [comment]
not against it, i just think it should have same customization options as active drops; i don't want to duplicate everything though... i need to think about it

**Comment by michaelowens** [comment]
Ah, I misunderstood then. Yeah, there's no rush behind this, it was just the first thing that popped into my head when I start using it and thought maybe get a boll rolling here on seeing how you guys think about it.

If you have any ideas I'd love to hear em and modify PR accordingly.

**Comment by Maxorq** [comment]
you can try to figure something out too... im not sure where to go with that

**Pull request closed without merging**